pkg/test/expectations/expectations.go

```diff
@@ -168,15 +169,27 @@ func ExpectDeleted(ctx context.Context, c client.Client, objects ...client.Objec
 	}
 }
 
-// TODO: Consider removing converting this into a standard reconciler; however, objects have to be properly updated if we don't rely on the Rconcile() method to get the object for us
+func ExpectSingletonReconciled(ctx context.Context, reconciler singleton.Reconciler) reconcile.Result {
```

Comment:
Can't this be done for us in operatorpkg/

Reply:
Yes and no. We're going to have to take a separate PR to migrate over all of these functions into operatorpkg. As it is, that's going to introduce a lot of change into the PR and we can't only do half of the changes at once because you're not allowed to define the same functions from two different packages when you are using "dot" importing